src/components/FindWallet/WalletTable.tsx

```diff
@@ -250,11 +251,34 @@ const FlexInfo = styled.div`
   }
 `
 
+const fadeIn = keyframes`
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+`
+
+const fadeOut = keyframes`
+  0% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 0;
+  }
+`
+
 const FlexInfoCenter = styled(FlexInfo)`
   justify-content: center;
   cursor: pointer;
   height: 100%;
   display: flex;
+  animation: ${fadeIn} 0.5s;
+
+  &.fade {
+    animation: ${fadeOut} 0.5s;
```

Comment:
It's a little too much. Maybe try half of this?

```suggestion
  animation: ${fadeIn} 0.25s;

  &.fade {
    animation: ${fadeOut} 0.25s;
```

Reply:
I was thinking of shortening the time to something like this, will bring this in :)